### DISCUSSION TITLE
Wrong variable interpolated in log message (instead of giving the hash of the notebook, the `Base.hash` is printed)

### DISCUSSION THREAD

**Discussion by Krastanov** [JuliaPluto/PlutoSliderServer.jl#25] [Issues] [answered]
![image](https://user-images.githubusercontent.com/705248/134939834-be581254-4917-43ce-ae52-f8489bf8b9b8.png)

This also seems to prevent me from knowing the url at which the notebook is served.

**Comment by fonsp** [comment, marked as answer]
Thanks! Fixed in https://github.com/JuliaPluto/PlutoSliderServer.jl/commit/ee66d4c541160ad6970c3d141625934b77efdb59
